workos.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   }
 
   spec.files = `git ls-files`.split($INPUT_RECORD_SEPARATOR)
+  spec.executables = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
```

Comment:
If we are keepinghttps://github.com/workos/workos-ruby/commit/e5f3c9bacc39801079b4028e637e95775ce80cef, this line not be getting added back, right? 🤔

Reply:
ugh, i forgot to set the base on my PR when i submitted and i wonder if changing it retroactively screwed stuff up. i'm going to close this and open a new one (i don't have this addition locally).